doc/features.rst

```diff
@@ -260,6 +260,7 @@ Type                              Notes
                                   time.
 `sphinxcontrib-svgbob`_           Supported
 `sphinxcontrib-youtube`_          Supported
+`sphinxcontrib-video`_            Supported
```

Comment:
(optional) Ideally, we try to these by name if possible; so this new entry should be before `sphinxcontrib-youtube`. Same with other documentation/implementation placement.